### PR TITLE
CORE-8621 Refactor DiskResources module Part 6

### PR DESCRIPTION
**Pull request by ash3rz**
NOTE: This is a branch on top of a branch.  The commits for just this PR start at `CORE-8621 Move Avu bean manipulation to util class`

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/search/cells/DiskResourceQueryFormDefaultAppearance.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    }
+
+    @Override
+    public int columnFormWidth() {
+        return 600;
+    }
+
+    @Override
+    public int columnWidth() {
+        return ((columnFormWidth()- 30) / 2) - 12;
```
Comment (psarando) [inline de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/search/cells/DiskResourceQueryFormDefaultAppearance.java]: ✨

File: de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataUtil.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.AstroThesaurusLoadConfig;
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.AstroThesaurusProxy;
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.MetadataTermLoadConfig;
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.MetadataTermSearchProxy;
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.OntologyLookupServiceLoadConfig;
+import org.iplantc.de.diskResource.client.presenters.metadata.proxy.OntologyLookupServiceProxy;
+import org.iplantc.de.diskResource.client.views.search.MetadataTermSearchField;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
```
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataUtil.java]: https://stackoverflow.com/questions/6382473/singleton-in-gwt-project. You might want to check this thread about using Singleton in GWT.

Comment (ash3rz) [reply de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataUtil.java]: Thanks!  I looked at the link and I think I'm implementing this singleton correctly so we can also use dependency injection, but if I've missed something please let me know.  The issue seemed to be regarding the use of `new SingletonClass()`, but I'm not doing that anywhere.

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**